content/rules/bridges.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Bridges
+---
+
+# Bridges
```

Comment:
We should probably have a section that at least links to https://docs.vmst.io/rules/threads, if not summarize our position on Threads here.

Reply:
It can just be something like the following after Nostr
```
## Threads

For information about our federation policies with Meta's Twitter-like service, Threads, see https://docs.vmst.io/rules/threads.
```

(I can't remember the Docus way to link, sorry. Too lazy to look it up right now.)